apis/nucleus/src/components/listbox/components/grid-list-components/grid-list-components.jsx

```diff
@@ -103,7 +108,7 @@ export default function getListBoxComponents({
         height={listHeight}
         width={width}
         itemCount={listCount}
-        itemData={{ ...commonItemData, listCount }}
+        itemData={{ ...commonItemData, listCount, contentFontStyle }}
```

Comment:
This is need for the `Grid` component too I suppose (i.e. the other component exported from this file).